wordpress-mobile/WordPress-Android
# Test PR for enabling pullquote in prod

This PR is for verifying Pullquote in prod. 
To test:

Create new article, add pullquote block. Verify placeholder shows when empty, and new line works as expected. 

PR submission checklist:
